### ISSUE TITLE
Add option for quiet (unobtrusive) message signing (no armor wrapping text)

### ISSUE THREAD

**Issue by stevemcquaid** [mailvelope/mailvelope#347]
I am unable to add pgp signing in official/business-related email chains because the armour wrapping text is rather obstructive of message readability as shown below. Personally I would like to use mailvelope rather silently at first, by simply attaching the signature to my messages/as text at the bottom of the message. the idea being that the PGP text does not interfere with a common person's ability to interpret the content of my messages.

I think the option to allow quieter signatures would allow a wider user base to adopt encryption.

I am happy to develop this feature in my spare time, might need some help architectural help though.

Example of obstructive signature:

-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Are you even able to read this clearly
without some searching in the message?

Regards,
Steve McQuaid
555.555.5555

-----BEGIN PGP SIGNATURE-----
Version: Mailvelope v1.2.3
Comment: https://www.mailvelope.com

<SIGNATURE....ETC>

**Comment by dashohoxha** [comment]
I totally agree with this request.
Is it somehow related to this one: https://github.com/mailvelope/mailvelope/issues/41 ?

**Comment by stevemcquaid** [comment]
@dashohoxha Yes - exactly. Though I think the title of the #41 should be changed. ("One up for reading or opening PGP/MIME." - @sigmun) haha

**Comment by toberndo** [comment]
This would require to add the signature file as an attachment to the email. Basically a similar process when you are currently using the file encryption feature of Mailvelope. Would be interesting to see if this process could be also automated like the other provider specific logic in https://github.com/mailvelope/mailvelope/blob/master/src/content-scripts/providerSpecific.js
